<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="principal">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QFrame" name="imagenes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(171, 187, 159);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="btn_mostrar_img1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>280</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(111, 139, 148)</string>
        </property>
        <property name="text">
         <string>Seleccionar imagen 1</string>
        </property>
       </widget>
       <widget class="QLabel" name="lb_imagen1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>280</width>
          <height>200</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 235, 169);</string>
        </property>
        <property name="text">
         <string>imagen 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lb_imagen2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>280</width>
          <height>200</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 235, 169);
</string>
        </property>
        <property name="text">
         <string>imagen 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_mostrar_img2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>510</y>
          <width>280</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(111, 139, 148)</string>
        </property>
        <property name="text">
         <string>Seleccionar imagen 2</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_gris">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>550</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Resultado en escala de grises</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="opciones">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>481</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
    background-color: rgb(221, 201, 150);
}

QPushButton{
	background-color: rgb(255, 179, 10);
}

QPushButton:hover {
	background-color: rgb(255, 120, 10);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGroupBox" name="basicas">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>161</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	background-color: rgb(217, 209, 194);
	color: rgb(0, 0, 0);
}

QPushButton{
	background-color: rgb(187, 184, 175);
	border-radius: 20px;
}

QPushButton:hover {
	color: rgb(255,255,255);
	background-color: rgb(97, 105, 83);
}</string>
        </property>
        <property name="title">
         <string>Operaciones Basicas</string>
        </property>
        <widget class="QPushButton" name="btn_suma">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Suma</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_resta">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Resta</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_multiplicacion">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Multiplicacion</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_division">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Divis√≥n</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="btn_negativo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>221</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Negativo</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="logicas">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>221</width>
          <height>161</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	background-color: rgb(197, 223, 224);
	color: rgb(0, 0, 0);
}

QPushButton{
	background-color: rgb(141, 174, 195);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(41, 174, 195);
}</string>
        </property>
        <property name="title">
         <string>Operaciones Logicas</string>
        </property>
        <widget class="QPushButton" name="btn_and">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>and</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_or">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>or</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_xor">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>xor</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_not">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>not</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="geometricas">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>221</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	background-color: rgb(189, 234, 219);
	color: rgb(0, 0, 0);
}

QPushButton{
	background-color: rgb(221, 234, 221);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(173, 234, 218);
}</string>
        </property>
        <property name="title">
         <string>Geometricas</string>
        </property>
        <widget class="QPushButton" name="btn_rotacion">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Rotacion</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_traslacion">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Traslacion</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_escalado">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Escalado</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_inclinacion">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Inclinacion</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_espejo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Espejo</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="histogramas">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>221</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	background-color: rgb(32, 198, 122);
	color: rgb(0, 0, 0);
}

QPushButton{
	background-color: rgb(157, 192, 157);
	border-radius: 20px;
}

QPushButton:hover {
	color: rgb(255,255,255);
	background-color: rgb(63, 136, 128);
}</string>
        </property>
        <property name="title">
         <string>Histograma</string>
        </property>
        <widget class="QPushButton" name="btn_hist_BN">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>BN</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_hist_color">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="Umbrales">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>221</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	background-color: rgb(131, 156, 181);
	color: rgb(0, 0, 0);
}

QPushButton{
	background-color: rgb(177, 225, 233);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 225, 233);
}</string>
        </property>
        <property name="title">
         <string>Umbrales</string>
        </property>
        <widget class="QPushButton" name="btn_umbral">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Umbral</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_umbral_Invertido">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Umbral Invertido</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_u_porNivel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Umbral por nivel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_u_porNivel_Invertido">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Umbral por nivel invertido</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_u_por2puntos">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Umbral por 2 puntos</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_u_por2puntos_Invertido">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Umbral por 2 puntos invertido</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="filtros">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>350</y>
          <width>221</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	background-color: rgb(238, 174, 170);
	color: rgb(0, 0, 0);
}

QPushButton{
	background-color: rgb(218, 174, 170);
	border-radius: 20px;
}

QPushButton:hover {
	color: rgb(255,255,255);
	background-color: rgb(178, 174, 170);
}</string>
        </property>
        <property name="title">
         <string>Filtros</string>
        </property>
        <widget class="QPushButton" name="btn_filtro_media">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Media</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_filtro_mediana">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>201</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Mediana</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
